test/torch/test_hook.py

```diff
@@ -72,9 +68,7 @@ def test_build_get_child_type():
 
 
 @pytest.mark.parametrize("attr", ["abs"])
-def test_get_pointer_unary_method(attr):
-    hook = syft.TorchHook(torch, verbose=True)
-
+def test_get_pointer_unary_method(attr, hook):
     instance_id = str(int(10e10 * random.random()))
 
     bob = syft.VirtualWorker(id=f"bob{instance_id}", hook=hook, is_client_worker=False)
```

Comment:
You can also use the same mecanism to load the workers (see in conftest the workers fixture)
This way it will be even more compact :)

Reply:
Oh yeah. On it!

Reply:
Wondering same can be done to other tests (files) ?

Reply:
Alot of them already have been. Feel free to update any that you see

Reply:
Would be super cool to have this done in all the files! 💯

Reply:
Will do it! :D

Reply:
`hook` as arg is not required when `workers` is passed. Changes have been made